lua/lang/scala.lua

```diff
@@ -0,0 +1,80 @@
+local M = {}
+
+M.config = function()
+  O.lang.scala = {
+    metals = {
+      active = false,
+      server_version = "0.10.2+33-c6441eb4-SNAPSHOT",
+      excluded_packages = {
+        "akka.actor.typed.javadsl",
+        "com.github.swagger.akka.javadsl",
+        "akka.stream.javadsl",
+      },
```

Comment:
I think, default list can be empty and user can exclude packages as per their needs.